src/main/java/com/google/sps/servlets/ContestsServlet.java

```diff
@@ -62,30 +64,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     Election election = Election.fromEntity(electionEntityOptional.get());
-    
-    List<String> contestJsonList =
+
+    List<JsonElement> contestJsonList =
```

Comment:
Can you put the JsonParser.parseString on a separate map() call?

Reply:
done